src/modules/tests/pass-completion/pass-completion.cat-be.reducer.ts

```diff
@@ -0,0 +1,16 @@
+
+import { PassCompletion } from '@dvsa/mes-test-schema/categories/BE';
+import { Action } from '@ngrx/store';
+
+export const initialState: PassCompletion = {
+  provisionalLicenceProvided: false,
+  passCertificateNumber: '',
+  code78Present: false,
+};
+
+export function passCompletionCatBEReducer(state: PassCompletion = initialState, action: Action) {
```

Comment:
missing return type and should be Cat Be not Cat BE

Reply:
Changed it